### DISCUSSION TITLE
Can QLMarkdown control the default "Quick Look" window size?

### DISCUSSION THREAD

**Discussion by setanarut** [sbarex/QLMarkdown#42] [Q&A] [answered]
Can QLMarkdown control the default "Quick Look" window size? masOS opens a larger window for html files. I noticed that some tables don't fit in the window (emoji tables).

**Comment by sbarex** [comment, marked as answer]
Hi, it is not possible to change the size of the window.

I seem to notice that the system remembers the last size set for each type of preview generator (so the previews obtained with the old API will have different sizes from those generated with the new ones)

**Comment by sbarex** [comment]
Implemented on release 1.0.9.
